website/app/components/doc/page/footer.hbs

```diff
@@ -21,6 +21,9 @@
       <li class="doc-page-footer__nav-item">
         <a href="https://www.hashicorp.com/privacy" target="_blank" rel="noopener noreferrer">Privacy</a>
       </li>
+      <li class="doc-page-footer__nav-item">
+        <a href="https://go.hashi.co/hds-feedback" target="_blank" rel="noopener noreferrer">Helios Feedback</a>
```

Comment:
One says `Helios Feedback` the other `Share feedback`. Probably better to standadize them.
```suggestion
        <a href="https://go.hashi.co/hds-feedback" target="_blank" rel="noopener noreferrer">Share feedback</a>
```